src/formatPkg.ts

```diff
@@ -634,3 +638,17 @@ function getModuleTypes(pkg: NicePackageType): ModuleType[] {
 
   return [...moduleTypes];
 }
+
+function getStyleTypes(pkg: NicePackageType): StyleType[] {
+  // style not declared - we will detect it later based on file list
+  if (!pkg.style) {
+    return [];
+  }
+
+  const ext = pkg.style.split('.').pop() as StyleType;
+  if (ext && styleFileExtensions.includes(ext)) {
```

Comment:
maybe instead of detecting specific extensions, we should save _all_ extensions?

Reply:
the idea was to filter out possible "trash" in the `style` field if the author set non-style file there. Not sure if it's necessary but it's just a preventive action here.

Reply:
That's an interesting idea but without the list of extensions of specific types, we wouldn't be able to set the `none` value - which I think provides some convenience, searching for `NOT none` vs `js OR cjs OR esm` - but I suppose you could also argue that it's better the user specifies the extensions they want explicitly.

Reply:
I can see some additional possibilities it would bring, e.g. searching for packages with web fonts, but there's again the question, if we want to make it low level and let users specify all extensions, or if we give them prepared categories (which could be done for fonts too).

Reply:
you can deal with none by defaulting to `styleType: ['none']`, as that can be filtered for, although then you're overlapping between possibly genuine extension called "none" (likely a non-issue)

Reply:
Oh, I misunderstood you meant just for the `style` field (thought you wanted a more generic version that would have all extensions of all files in the package) - since we combine the information from `style` with the information based on file listings, I think it makes sense to have a static set of expected values. If we find there's a new value that makes sense, it should be added to the file listing detection as well.

Reply:
how would we find out which other extensions could make sense? maybe `.styl` has some usage, or something new would come up. If we base it on the data instead of a static list, we won't miss any extensions in the future if they become popular. What do you think?

Reply:
Ok, that's a fair point - let's allow all extensions for the `style` field, just normalize it to lowercase.

Reply:
fixed here https://github.com/algolia/npm-search/pull/683/commits/ab397697220d928bfce2b8013ddc54c17f1d6592